Apps/W1/File - Azure BLOB Storage Connector/app/src/BlobStorageAccountWizard.Page.al

```diff
@@ -0,0 +1,161 @@
+                NotBlank = true;
+
+                trigger OnValidate()
+                begin
+                    IsNextEnabled := BlobStorageConnectorImpl.IsAccountValid(Rec);
+                end;
+            }
+
+            field(StorageAccountNameField; Rec."Storage Account Name")
+            {
+                ApplicationArea = All;
+                Caption = 'Storage Account Name';
+                ToolTip = 'Specifies the Azure Storage name.';
+                ShowMandatory = true;
+
+                trigger OnValidate()
+                begin
+                    IsNextEnabled := BlobStorageConnectorImpl.IsAccountValid(Rec);
+                end;
+            }
+
+            field(PasswordField; Password)
+            {
+                ApplicationArea = All;
+                Caption = 'Password';
+                ExtendedDatatype = Masked;
+                ToolTip = 'Specifies the shared key of the Storage Blob.';
+                ShowMandatory = true;
+            }
+
+            field(ContainerNameField; Rec."Container Name")
+            {
+                ApplicationArea = All;
+                Caption = 'Container Name';
+                ToolTip = 'Specifies the container to use of the Storage Blob.';
+                ShowMandatory = true;
+
+                trigger OnLookup(var Text: Text): Boolean
+                var
+                    BlobStorageConnectorImpl: Codeunit "Blob Storage Connector Impl.";
+                begin
+                    BlobStorageConnectorImpl.LookUpContainer(Rec, Password, Text);
+                    exit(true);
+                end;
+
+                trigger OnValidate()
+                begin
+                    IsNextEnabled := BlobStorageConnectorImpl.IsAccountValid(Rec);
+                end;
+            }
+        }
+    }
+
+    actions
+    {
+        area(processing)
+        {
+            action(Back)
+            {
+                ApplicationArea = All;
+                Caption = 'Back';
+                ToolTip = 'Back';
+                Image = Cancel;
+                InFooterBar = true;
+
+                trigger OnAction()
+                begin
+                    CurrPage.Close();
+                end;
+            }
+
+            action(Next)
+            {
+                ApplicationArea = All;
+                Caption = 'Next';
+                Image = NextRecord;
+                Enabled = IsNextEnabled;
+                InFooterBar = true;
+                ToolTip = 'Next';
+
+                trigger OnAction()
+                begin
+                    BlobStorageConnectorImpl.CreateAccount(Rec, Password, BlobStorageAccount);
+                    CurrPage.Close();
+                end;
+            }
+        }
+    }
+
+    var
+        BlobStorageAccount: Record "File Account";
+        MediaResources: Record "Media Resources";
+        BlobStorageConnectorImpl: Codeunit "Blob Storage Connector Impl.";
+        [NonDebuggable]
+        [InDataSet]
```

Comment:
Not sure, but are these still needed? I thought InDataSet was no longer used in web clients.